tf_fastmri_data/dataset_builder.py

```diff
@@ -118,18 +137,13 @@ def _build_datasets(self):
                 num_parallel_calls=self.num_parallel_calls,
                 deterministic=True,
             )
-        if self.brain:
+        if self.save_shapes:
             output_shape_ds = tf.data.Dataset.from_tensor_slices(
                 [load_output_shape_from_file(f) for f in self.filtered_files],
             )
             self._raw_ds = tf.data.Dataset.zip(
                 (self._raw_ds, output_shape_ds)
             )
-            self._raw_ds = self._raw_ds.map(
```

Comment:
Why did you remove this?

Reply:
Oh this code was broken, I actually wanted to ask you this. This code from what i see tries to open dataset so that we have it as (kspace, image, shape) right? That is not happening. In fact the code errors out right here. 
I think i am missing something with respect to the aim of this dataset map.

Reply:
Can you tell me exactly the code you ran to see a failure?
I tried the following:

```python
from tf_fastmri_data.datasets.cartesian import CartesianFastMRIDatasetBuilder

ds_builder = CartesianFastMRIDatasetBuilder(
    dataset='val',
    brain=True,
    multicoil=True,
)

ds = ds_builder.preprocessed_ds

res = next(iter(ds))
```

without errors

Reply:
Lol, I think I messed my environments, This is only causing issues for me right now.. Reverting this